src/main/resources/anniv5Resources/localization/eng/gemspack/Keywordstrings.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "PROPER_NAME": "Gem",
+    "NAMES": [
+      "gem"
+    ],
+    "DESCRIPTION": "At #yRest #ySites, you may spend 20 Gold to permanently remove a #yGem as a free action, then permanently #yModify a card in deck, the chosen card permanently gaining the #yGem's effects."
```

Comment:
Lot of permanent in this sentence.
Could say something like "At Rest Sites, you may spend 20 Gold to socket a Gem into a card as a free action, the card gains the gem's effect and the gem is permanently gone."